### PR TITLE
Add a rake task and configs to harvest MIT content from their …

### PR DESCRIPTION
**Pull request by jkeck**
…GeoBlacklight instance.

Part of #619

We can ship this once OpenGeoMetadata/GeoCombine#94 is merged

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -1,6 +1,6 @@
 GIT
   remote: https://github.com/OpenGeoMetadata/GeoCombine.git
-  revision: 77fde5526c0d1c4d9b6c79d58cd0acb7351ce46b
+  revision: b2688d4a75696e2211ae38f98989fe1a1c06542c
```
Comment (mejackreed) [inline Gemfile.lock:3]: GeoCombine v0.5.0 has just been :shipit:

Comment (jkeck) [reply Gemfile.lock:3]: Okay, do you want us to get on a released version?

Comment (mejackreed) [reply Gemfile.lock:3]: Yeah we might as well

**Comment by jkeck** [comment]
FWIW, I deployed this to stage and ran it (while temp. disabling SSL verification).  We have the [10,005 MIT records](https://kurma-earthworks-stage.stanford.edu/?f%5Bdct_provenance_s%5D%5B%5D=MIT) in stage.

**Pull request merged**